app/models/sorting/event.js

```diff
@@ -0,0 +1,28 @@
+const getUnixDate = (dateString) => {
+  return new Date(dateString).getTime()
+}
+
+const sortEventsDescCompareFn = (a, b) => {
```

Comment:
I was thinking this could be simplified to something like:
  const sortValueA = a.activity?.activityDate ? a.activity.activityDate : a.timestamp
  const sortValueB = b.activity?.activityDate ? b.activity.activityDate : b.timestamp
  return sortValueB - sortValueA
  
  Would that not work?

Reply:
✓ should sort two none activities correctly
      ✓ should sort two activities correctly
      ✕ should sort two activities correctly given same date
      ✓ should sort two activities correctly given different types (1 ms)
      ✕ should sort two activities correctly given Date is same

Reply:
it doesn't take into account the case if both items have the same activityDate, but different timestamps

Reply:
Ah, ok, hadn't thought of that scenario. Ok, leave as is then